packages/grafana-e2e/src/components/index.ts

```diff
@@ -0,0 +1,7 @@
+import { TestDataQueryEditor } from './queryEditors/testdata';
+
+export const Components = {
+  QueryEditors: {
+    TestData: TestDataQueryEditor,
+  },
+};
```

Comment:
What are your thoughts about the Pages structure? I find it helpful, but I'm thinking these index files are becoming a nuisance at the same time. Also, if we change the pageFactory to include pageObjects then I see no need for the Components node.